### PR TITLE
Docs(Turbo): Update to guide for uploading with Turbo

### PR DESCRIPTION
**Pull request by Bobinstein**
Small update to guide for uploading with Turbo, includes adding tags to files during upload. Also added the page to sidebar.

### CODE REVIEW

File: docs/src/guides/posting-transactions/turbo.md [hunk 1]
```diff
@@ -101,14 +106,21 @@ console.log("Result", uploadResult);
...
+	signal: AbortSignal.timeout(10_000), // Optional: cancel the upload after 10 seconds
+  dataItemOpts // Optional
 });
 console.log(JSON.stringify(uploadResult, null, 2));
 ```
 
 ## Resources
 
 - Dive into the [Code](https://github.com/ardriveapp/turbo-sdk)
+- Join the discussion in the [ArDrive Discord](https://discord.com/invite/ya4hf2H)
```
Comment (atticusofsparta) [inline docs/src/guides/posting-transactions/turbo.md:126]: Was this a permanent invite link?

File: docs/src/guides/posting-transactions/turbo.md [hunk 2]
```diff
@@ -101,14 +106,21 @@ console.log("Result", uploadResult);
...
+    tags: [{name: 'test', value: 'test'}] // add tags
+  }
 const uploadResult = await turbo.uploadFile({
 	fileStreamFactory: () => fs.createReadStream(filePath),
 	fileSizeFactory: () => fileSize,
-	signal: AbortSignal.timeout(10_000), // cancel the upload after 10 seconds
+	signal: AbortSignal.timeout(10_000), // Optional: cancel the upload after 10 seconds
+  dataItemOpts // Optional
 });
 console.log(JSON.stringify(uploadResult, null, 2));
```
Comment (atticusofsparta) [inline docs/src/guides/posting-transactions/turbo.md:120]: Good practice to add a code snippet in the `console` or `bash` language with the expected output here

File: docs/src/guides/posting-transactions/turbo.md [hunk 3]
```diff
@@ -67,14 +67,19 @@ const JWK = JSON.parse(fs.readFileSync(process.env.PATH_TO_WALLET).toString());
...
+    tags: [{name: 'test', value: 'test'}] // add tags
+  }
   const uploadResult = await turbo.uploadFile({
     fileStreamFactory: () => fs.createReadStream(filePath),
     fileSizeFactory: () => fileSize,
-    signal: AbortSignal.timeout(10_000), // cancel the upload after 10 seconds
+    signal: AbortSignal.timeout(10_000), // Optional: cancel the upload after 10 seconds
+    dataItemOpts // Optional
   });
   console.log(JSON.stringify(uploadResult, null, 2));
```
Comment (atticusofsparta) [inline docs/src/guides/posting-transactions/turbo.md:82]: exemplify the output in code snippet

File: docs/src/guides/posting-transactions/turbo.md (hunk 1 above)
Comment (Bobinstein) [reply docs/src/guides/posting-transactions/turbo.md:126]: Yes, this is the same permanent link used on the ArDrive docs portal

File: docs/src/guides/posting-transactions/turbo.md (hunk 2 above)
Comment (Bobinstein) [reply docs/src/guides/posting-transactions/turbo.md:162]: Added

**Comment by rosmcmahon** [comment]
any reason why this isn't reviewed and merged?

**Review by rosmcmahon — APPROVED** [review]

**Pull request merged**